api/src/domain-logic/supabase.js

```diff
@@ -0,0 +1,8 @@
+const { createClient } = require('@supabase/supabase-js')
+
+const url = process.env.SUPABASE_URL
+const anonKey = process.env.SUPABASE_ANON_KEY
+
+const supabase = createClient(url, anonKey)
```

Comment:
`const supabase = createClient(url, anonKey, { shouldThrowOnError: true })`

To throw error rather than returning error property.